devtools/bom-descriptor-json/src/main/resources/extensions-overrides.json

```diff
@@ -372,6 +372,12 @@
       "artifact-id": "camel-quarkus-pdf",
       "version": "0.3.1",
       "description": "Camel PDF support"
+    },
+    {
+      "group-id": "org.kie.kogito",
+      "artifact-id": "kogito-quarkus",
+      "version": "0.8.0",
+      "description": "Kogito extension"
```

Comment:
Why this is still needed if kogito is being removed from the Quarkus codebase?

Reply:
it will pickup what is in the extension - which btw. should be more than just "Kogito extension" :)